newsroom/wire/views.py

```diff
@@ -145,6 +144,8 @@ def versions(_id):
 @blueprint.route('/wire/<_id>')
 def item(_id):
```

Comment:
If the users have the links to the killed or updated items, will we still return it?

Reply:
right now yes. if the item is killed it will have the kill notice, but if an update is killed or previous version it will work as usual